src/nexgen/nxs_utils/Detector.py

```diff
@@ -137,6 +145,30 @@ def hasMeta(self) -> bool:
         return True
 
 
+@dataclass
+class CetaDetector(DataClassJsonMixin):
+    """Define a Ceta-D detector."""
+
+    description: str
+    image_size: List[float] | Tuple[float]
+    pixel_size: List[str | float] = field(
+        default_factory=lambda: ["0.014mm", "0.014mm"]
+    )
+    sensor_material: str = "Si"
+    sensor_thickness: str = "0.00000000000001mm"
```

Comment:
10^-14 mm doesn't sound right for a sensor thickness. Docs seem to point at something around 14um, could it be something similar?

Reply:
Yes, I fixed this. This is probably not used anywhere so it won't cause any problems. I think some parameter 'mu' is computed from this, but it is not used.